api/external/applications/applications.proto

```diff
@@ -18,44 +18,42 @@ import "api/external/common/query/parameters.proto";
 service ApplicationsService {
   rpc GetServiceGroups(ServiceGroupsReq) returns (ServiceGroups) {
     option (google.api.http).get = "/beta/applications/service-groups";
-    // TODO (dan, 2/2019): need to replace this once we have resources and such
-    // created in the auth system
     option (chef.automate.api.policy) = {
-      resource: "nodes"
+      resource: "service_groups"
       action: "list"
     };
     option (chef.automate.api.iam.policy) = {
-      resource: "infra:nodes"
-      action: "infra:nodes:list"
+      resource: "applications:serviceGroups"
+      action: "applications:serviceGroups:list"
     };
   };
   rpc GetServiceGroupsHealthCounts(ServiceGroupsHealthCountsReq) returns (HealthCounts) {
     // TODO (afiune, 3/2019): need to replace this once we have resources and such
     // created in the auth system
     option (google.api.http).get = "/beta/applications/service_groups_health_counts";
-    option (chef.automate.api.policy).resource = "nodes";
+    option (chef.automate.api.policy).resource = "service_groups";
     option (chef.automate.api.policy).action = "list";
-    option (chef.automate.api.iam.policy).resource = "infra:nodes";
-    option (chef.automate.api.iam.policy).action = "infra:nodes:list";
+    option (chef.automate.api.iam.policy).resource = "applications:serviceGroups";
+    option (chef.automate.api.iam.policy).action = "applications:serviceGroups:list";
   };
   rpc GetServices(ServicesReq) returns (ServicesRes) {
     // TODO (afiune, 4/2019): need to replace this once we have resources and such
     // created in the auth system
     option (google.api.http).get = "/beta/applications/services";
-    option (chef.automate.api.policy).resource = "nodes";
+    option (chef.automate.api.policy).resource = "service_groups";
```

Comment:
I don't think this endpoint has anything to do with service-groups here. This is only listing all services so we would probably need a different resource?

Reply:
For any future readers of this, we decided to stick with `service_groups` because this matches the labeling in the UI, so that is what users will expect to see.